extensions/content-manager/admin/src/components/CustomTable/Row.js

```diff
@@ -84,6 +85,38 @@ function Row({ goTo, isBulkable, row, headers, level, relationColumnName }) {
 
   const { emitEvent } = useGlobalContext();
 
+  const generateIcoContainerIcons  = () => {
+    let baseIcons = [
+      {
+        icoType: 'pencil-alt',
+        onClick: () => {
+          emitEvent('willEditEntryFromList');
```

Comment:
I would rather use const instead of string, like actions in redux,

Reply:
Strapi code